build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/rbs/dynamic_error_list/sqs.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This file is a dynamically defined error class written by hand.

Reply:
Hmm. Is the intention to generate possible DynamicErrors? I think we'd rather everything be code generated against the model and not anything manual. For cases not in the model, we should reference the `Aws::<Service>::Errors::ServiceError` parent class? We know the methods on this class. It's not a perfect experience but rescue of the parent will succeed for these errors.

Reply:
Makes sense.. We have over 300 services, so maintaining a list of un-modeled services seems like a nightmare and could get stale fast.. I think we should default dynamic errors these to the parent class since they share members (I think this is the useful part). We are pushing on service teams to model errors more extensively.

Reply:
Yeah -  maintaining a hand written list of errors across 300+ services is untenable (and theres no actual source of truth).  We are stuck with the dynamic errors approach within this major version to avoid breaking users, but its far from ideal.  We are pushing service teams to correctly model errors on their APIs.  I think I'd agree with @mullermp  that these should just use the `Aws::<Service>::Errors::ServiceError` parent class, though I know its not an ideal experiance.